test/test_manifest.json

```diff
@@ -5907,5 +5907,36 @@
        "link": true,
        "forms": true,
        "type": "eq"
+    },
+    {  "id": "issue14021",
+       "file": "pdfs/issue14021.pdf",
+       "md5": "d18aa84135ce985c70a8f56306ecb95f",
+       "link": true,
+       "rounds": 1,
+       "firstPage": 2,
+       "lastPage": 2,
+       "type": "eq",
+       "forms": true
+    },
+    {  "id": "issue14021-1",
```

Comment:
Let's name it "issue14021-storage" instead, since that seems slightly more helpful.